src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -156,13 +156,24 @@ else if (parseResult.isUsageHelpRequested()) {
         Runtime.getRuntime().exit(exitStatus);
     }
 
+    /**
+     * Prints version string when the user requests version help (--version or -V).
+     *
+     * @noinspection UseOfSystemOutOrSystemErr
+     * @noinspectionreason UseOfSystemOutOrSystemErr - driver class for Checkstyle requires
+     *      usage of System.out and System.err
+     */
+    private static void printVersionToSystemOutput() {
```

Comment:
Confirmed moving out as this version method doesn't work without jar. We need to exclude it from pitest.